internal/server/events.go

```diff
@@ -61,6 +73,61 @@ func (s *Server) eventsHandler() http.HandlerFunc {
 	}
 }
 
+// eventHandler returns a handler to get a specific event.
+func (s *Server) eventHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// Get new event data from TBA if event data is over 24 hours old
+		err := s.updateEvents()
```

Comment:
If a function only has an error return value, and you check it on the next line, I think it makes sense to collapse them together.